core/src/test/java/org/apache/james/core/UsernameTest.java

```diff
@@ -148,9 +149,9 @@ void fromUsernameShouldThrowWhenTooLong() {
     }
 
     @Test
-    void fromUsernameShouldThrowWhenMultipleDomainDelimiter() {
-        assertThatThrownBy(() -> Username.of("aa@aa@aa"))
-            .isInstanceOf(IllegalArgumentException.class);
+    void arbitraryCharInLocalPart() {
+        assertThatCode(() -> Username.of("Whatever_I- want3$$$###_! as lons I do not give a domain"))
+            .doesNotThrowAnyException();
```

Comment:
Why do you change a test for Username in this commit/PR? Mix up with something else maybe?

Reply:
Sorry mistake I likely wrote some tests to answer a ML thread while running long compilations